fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -35,20 +36,54 @@
  * @since 6.0.0
  */
 @Configuration
+@PropertySource (value = "file:${" + FedoraPropsConfig.FCREPO_CONFIG_FILE + "}", ignoreResourceNotFound = true)
 public class FedoraPropsConfig {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FedoraPropsConfig.class);
 
+    public static final String FCREPO_CONFIG_FILE = "fcrepo.config-file";
```

Comment:
For consistency, can we rename this property to not include a `-`. Maybe `fcrepo.config.file` or `fcrepo.config` or `fcrepo.config.location`... or something else?

Reply:
sure - no problem.